pom.xml

```diff
@@ -836,6 +836,12 @@
             <artifactId>commons-codec</artifactId>
             <version>1.15</version>
         </dependency>
+        <dependency>
```

Comment:
I don't remember I actually use this. If it used in Factions, it should provide by it.

Reply:
It's was used for compressing logs.